r-package/grf/R/plot.R

```diff
@@ -67,8 +67,9 @@ size = ", num_samples, '"];')
 #' Export a tree in DOT format.
 #' This function generates a GraphViz representation of the tree,
 #' which is then written into `dot_string`.
+#' Can be used to save the plot with the following command: cat(DiagrammeRsvg::export_svg(DiagrammeR::grViz(dot_string)), file=path)
```

Comment:
Could maybe be added as a comment in `plot.grf_tree ` instead?

And why does this function have to be exported? is it not enough to do:

```R
tree.plot = plot(tree)
cat(DiagrammeRsvg::export_svg(tree.plot), file='plot.svg')
```

Reply:
Thanks @erikcs, I read too quickly and didn't see that you could call `export_svg` on the 'graph viz' object itself. Your suggested approach of keeping `export_graphviz` private and documenting this in `plot.grf_tree` makes sense to me.